src/h3lib/include/h3api.h.in

```diff
@@ -610,6 +610,30 @@ DECLSPEC void H3_EXPORT(getH3UnidirectionalEdgeBoundary)(H3Index edge,
                                                          GeoBoundary *gb);
 /** @} */
 
+/** @defgroup cellToVertex cellToVertex
+ * Functions for cellToVertex
+ * @{
+ */
+/** @brief Returns a single vertex for a given cell, as an H3 index */
+DECLSPEC H3Index H3_EXPORT(cellToVertex)(H3Index origin, int vertexNum);
+/** @} */
+
+/** @defgroup cellToVertexes cellToVertexes
+ * Functions for cellToVertexes
+ * @{
+ */
+/** @brief Returns a single vertex for a given cell, as an H3 index */
```

Comment:
Should this be "array of vertexes"? And what do folks think about using the word "returns" when the function is `void` and doesn't *actually* return anything? Do we have a more accurate alternative? Maybe "writes out"?

Reply:
Whoops, copy/pasta error. Updated to `Returns all vertexes for a given cell, as H3 indexes` - "returns" seems to be what we use elsewhere, and makes more sense for the bindings (returning via modifiying the `out` array is basically an implementation detail).